### PR TITLE
Add text to TOS page

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd

### CODE REVIEW

File: templates/theme-settings.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## theme-settings.html
+
+<%!
+  theme_settings_vars = {
+    'effective_date': 'October 26, 2015',
+    'accessibility_email': '<a href="mailto:accessibility@stanfordonline.zendesk.com">accessibility@stanfordonline.zendesk.com</a>',
```
Comment (stvstnfrd) [inline templates/theme-settings.html]: It's a shame these aren't already pulled from `settings`..

**Comment by stvstnfrd** [comment]
Assuming this is just copy/pasta, +1

File: templates/theme-settings.html (hunk 1 above)
Comment (caesar2164) [inline templates/theme-settings.html]: yeah, but since this is just a temporary solution until Comp Theming, I don't think it;s worth fixing right now...

**Comment by caesar2164** [comment]
yep just copy & "pasta" (sic)

**Pull request merged**